src/styles/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap");
+ 
```

Comment:
I would suggest you change the name of the file on card.css or something like that 
style.css can confuse